### PR TITLE
Adds power boots to ert and deathsquaddies [WIP]

### PR DESCRIPTION
**Pull request by ghost**
🆑 ArmAndHammer
add: Added power boots!
add: they kick HARD, gamers. make sure they're on the ground for best effect!
add: they also kick through DOORS so they're great for breaching like a cool guy
/🆑

[why]: wyzack wants /betterbreaching/ with deathsquads and erts and i can't agree more. it's cool, it's in the code, and soon some fucker will add it to rnd. hell yeah

TO DO:
the item that grants these
adding that item to loadouts of ert
~~adding disarm kicks~~ (i like comboing with non-martial items so i cut it)
adding kicking down doors
removing awesine (DONE)

### CODE REVIEW

**Comment by Wyzack** [comment]
Will head-destroying curb stomp executions of critted people be a possibility? Make it take several seconds to line up for balance reasons of course.

**Comment by Armhulen** [comment]
that's the easiest part of this

**Comment by MrDoomBringer** [comment]
this pr:
image

**Comment by Armhulen** [comment]
@MrDoomBringer MY MAN

**Comment by Hatterhat** [comment]
HE'LL YEA

**Comment by Armhulen** [comment]
HE WILL YEA

I'LL HOORAY EMOJI TO THAT WOOOOOOOOOOo

**Comment by epochayur** [comment]
is there a cooldown on doorkicking?

**Comment by Armhulen** [comment]
I probably won't considering death squads have pulse rifles and same with amber ert

**Comment by Wyzack** [comment]
Literally just for style when these teams already have instant delete gun with infinite ammo. Might be a concern if a slightly neutered version is ever introduced as a crew or traitor item but that is a whole other balance related can of worms not worth discussing just yet

**Comment by Spaceinaba** [comment]
If the door doesn't break and slide multiple tiles into the room in a shower of sparks I will give you a downvote

**Comment by Spaceinaba** [comment]
also a dropkick that fuckin floors people would be super stylish

**Comment by Armhulen** [comment]
@SpaceInaba that's what I'm doing dummy no drop kick tho

**Comment by Spaceinaba** [comment]
*sob

**Comment by Buggy123** [comment]
>removing awesine

You look at your armhulen and sigh

**Comment by MrDoomBringer** [comment]
oranges y

**Comment by Bawhoppen** [comment]
Probably anti-sec creep stuff or something?

**Comment by kevinz000** [comment]
@MrDoomBringer get a discord or irc bouncer so i can chat about cargo things

**Comment by Armhulen** [comment]
calm down friends i got this

**Comment by Armhulen** [comment]
image

**Comment by Armhulen** [comment]
image

**Comment by Armhulen** [comment]
image

**Comment by Armhulen** [comment]
korrrrr grabbing a pizzaaaaaaaaaaaa yeah yeah yeah get hyped

**Comment by ShizCalev** [comment]
24 hours

**Comment by ayyourboibilly** [comment]
hi

**Comment by Armhulen** [comment]
just like that folks.

**Comment by Jordie0608** [comment]
You have a mix of space and tabs for indentation.

**Review by ShizCalev — CHANGES_REQUESTED** [review]
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#tabs-not-spaces

File: code/datums/martial/power_kick [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+/datum/martial_art/power_kick/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	return harm_pkick(A,D)
+
+/datum/martial_art/power_kick/proc/harm_pkick(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+  if(D.lying)
+  	if(D.stat && user.get_bodypart("head"))
+		var/obj/item/bodypart/head/Dhead = D.get_bodypart("head")
+		if(do_after(A, 50, D)) //if if if but i promise this is needed
+			Dhead.qdel() //this can't be right but wip wip wip wip
```
Comment (kevinz000) [inline code/datums/martial/power_kick]: qdel(Dhead)

Comment (kevinz000) [reply code/datums/martial/power_kick]: ALSO WHAT THE FUCK UNBALANCED

**Comment by Cyberboss** [comment]
>no dme change
>Not even a dm file

Try harder to do better

**Comment by Armhulen** [comment]
I JUTS FIXED THAT

wait fuck

**Comment by Armhulen** [comment]
okay i'm a braindead retard i'll remake this pr tonight with correct spacing and dme etc

**Comment by Armhulen** [comment]
but now i know why oranges ragequit

**Comment by CitrusGender** [comment]
what is wrong with all of you

**Comment by Armhulen** [comment]
because one day this will return

**Pull request closed without merging**